packages/scaffold/package.json

```diff
@@ -19,9 +19,7 @@
     "@web3modal/common": "4.1.5",
     "@web3modal/core": "4.1.5",
     "@web3modal/ui": "4.1.5",
-    "lit": "3.1.0"
-  },
-  "peerDependencies": {
+    "lit": "3.1.0",
```

Comment:
Was this a peerdep? Why move it into dependencies?

Reply:
comma change :)